### PR TITLE
Fix/do not accept containers with non unique attributes

### LINKED ISSUE #634: Node accepts containers with non unique attributes
The same as #633 but for container attributes.

```
$ ./bin/neofs-cli -r [...] --binary-key [...] container create --policy [...] --attributes Timestamp=0

$ ./bin/neofs-cli -r [...] --binary-key [...] container get --cid [...]
...
attribute: Timestamp=0 (1970-01-01 03:00:00 +0300 MSK)
attribute: Timestamp=1624378604 (2021-06-22 19:16:44 +0300 MSK)
```

### PR DESCRIPTION
**Pull request by carpawell**
Closes #634.

### CODE REVIEW

File: pkg/core/container/fmt.go [hunk 1]
```diff
@@ -32,5 +35,14 @@ func CheckFormat(c *container.Container) error {
 		return fmt.Errorf("incorrect nonce: %w", err)
 	}
 
+	// check if there are repeated attributes
+	uniqueAttr := make(map[string]struct{})
```
Comment (cthulhu-rider) [inline pkg/core/container/fmt.go:39]:
```suggestion
	uniqueAttr := make(map[string]struct{}, len(attrs))
```
and reuse `attrs := c.Attributes()` in `for`

File: pkg/core/container/fmt_test.go [hunk 2]
```diff
@@ -41,4 +41,18 @@ func TestCheckFormat(t *testing.T) {
 	c.SetNonceUUID(uuid.New())
 
 	require.NoError(t, CheckFormat(c))
+
+	// set same key attributes
+	attr1 := container.NewAttribute()
+	attr1.SetKey("same")
+	attr2 := container.NewAttribute()
+	attr2.SetKey("same")
```
Comment (cthulhu-rider) [inline pkg/core/container/fmt_test.go:49]:
```suggestion
	attr2.SetKey(attr1.Key())
```

File: pkg/core/container/fmt_test.go [hunk 3]
```diff
@@ -41,4 +41,18 @@ func TestCheckFormat(t *testing.T) {
...
+
+	attrs := container.Attributes{attr1, attr2}
+
+	c.SetAttributes(attrs)
+	require.ErrorIs(t, CheckFormat(c), errRepeatedAttributes)
+
+	attr2.SetKey("not same")
```
Comment (cthulhu-rider) [inline pkg/core/container/fmt_test.go:56]:
```suggestion
	attr2.SetKey(attr1.Key() + "smth")
```

File: pkg/core/container/fmt.go (hunk 1 above)
Comment (carpawell) [reply pkg/core/container/fmt.go]: added

File: pkg/core/container/fmt_test.go (hunk 2 above)
Comment (carpawell) [reply pkg/core/container/fmt_test.go]: changed

File: pkg/core/container/fmt_test.go (hunk 3 above)
Comment (carpawell) [reply pkg/core/container/fmt_test.go]: changed

**Comment by alexvanin** [comment]
Let's to that too: https://github.com/nspcc-dev/neofs-node/issues/634#issuecomment-866840027

**Review by cthulhu-rider — APPROVED** [review]

**Review by alexvanin — APPROVED** [review]

**Pull request merged**